runtime/doc/news.txt

```diff
@@ -39,6 +39,12 @@ The following new APIs or features were added.
 iterators |luaref-in|.
 • Added |vim.keycode()| for translating keycodes in a string.
 
+• Added automatic linting of treesitter query files (see |ft-query-plugin|).
+  Automatic linting can be turned off via >lua
+	vim.g.query_lint_on = {}
```

Comment:
```suggestion
	vim.g.query_lint_on = {}
<
```

(Vim's help parser is laissez-faire, but in our documentation we should strive for "proper" syntax to make the treesitter parser's job easier.)

Reply:
adopted the change as suggested